app/models/product.rb

```diff
@@ -21,6 +22,8 @@ class Product < ApplicationRecord
   validates :image_url, allow_blank: true, url: true
   validates :permalink, uniqueness: true, format: { with: PERMALINK_FORMAT }
   validates_length_of :words_in_permalink, minimum: 3
+  validates :category_id, presence: true
```

Comment:
This is not required as already handled by belongs_to

Reply:
Done